packages/web-twig/tests/Helper/ComponentsSnapshotHelper.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lmc\SpiritWebTwigBundle\Helper;
+
+use Lmc\SpiritWebTwigBundle\Helper\TwigHelper;
+use PHPUnit\Framework\TestCase;
+use Spatie\Snapshots\MatchesSnapshots;
+use Twig\Environment;
+
+class ComponentsSnapshotHelper extends TestCase
```

Comment:
maybe better will be create AbstractSnapshotTest class instead Helper and set `$SNAPSHOT_SOURCES = __DIR__ . '/__fixtures__';` as default and I think you don't need to overload this property for each tests..

Reply:
I think I tried this approach and the problem was that the `__DIR__` was transformed from the abstract class location than from the test class location.

Reply:
This problem still remains even after refactoring to the abstract class.

Reply:
Solve with a trick with a reflection class https://almamedia.slack.com/archives/C0502FE7ZSA/p1691575232858599?thread_ts=1691571229.187669&cid=C0502FE7ZSA